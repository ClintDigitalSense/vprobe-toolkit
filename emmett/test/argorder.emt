# make sure function definitions are able to emit args in non-alphabetical
# order. Sigh.

void t(int b, int a) { }
