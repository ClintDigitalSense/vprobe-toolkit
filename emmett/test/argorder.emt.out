(version 0.2)
(defun t(b a )( do  ))